tests/e2e/test_e2e_images.py

```diff
@@ -200,19 +200,24 @@ def test_images_delete(
     tag: str,
 ) -> None:
     image_no_tag = image.replace(f":{tag}", "")
-    image_full_str = f"image://{helper.cluster_name}/{helper.username}/{image_no_tag}"
+    image_full_str = f"image://{helper.cluster_name}/{helper.username}/{image}"
+    image_full_str_no_tag = (
+        f"image://{helper.cluster_name}/{helper.username}/{image_no_tag}"
+    )
 
     captured = helper.run_cli(["image", "ls", "-l", "--full-uri"])
     print("\n")
-    print(f"== {image_full_str} ==")
+    print(f"== {image_full_str_no_tag} ==")
     print(captured.out)
     print("=" * 20)
     matching_lines = [
-        line for line in captured.out.splitlines() if image_full_str == line.split()[0]
+        line
+        for line in captured.out.splitlines()
+        if image_full_str_no_tag == line.split()[0]
     ]
     assert len(matching_lines) == 1
 
-    helper.run_cli(["image", "rm", image])
+    helper.run_cli(["image", "rm", image_full_str])
 
     captured = helper.run_cli(["image", "ls", "-l", "--full-uri"])
     matching_lines = [
```

Comment:
I'm not sure that this check is good enough. You are only removing single tag, image repo (`image://default/neuro-e2e-user/e2e-banana-image` is content of `image_full_str`) can be still in the output of `neuro image ls` if tags generated by other tests are still there.

Reply:
Right, image deletion works for single tag, that's expected